{ 'test_2': { 'left_arm': [],
              'left_leg': [ { 'duration': 1.91,
                              'name': u'high_step_4_1_left',
                              'positions': [ 0.01,
                                             -0.1649,
                                             -1.4576,
                                             1.5538,
                                             0.1635,
                                             -0.0629],
                              'starttime': 0.0}],
              'neck': [],
              'right_arm': [],
              'right_leg': [ { 'duration': 1.91,
                               'name': u'high_step_0_right',
                               'positions': [ 0.0225,
                                              -0.1666,
                                              -0.0149,
                                              0.0022,
                                              0.0094,
                                              -0.1544],
                               'starttime': 0.0}],
              'torso': []}}
